aodndata/srs/srs_surface_waves.py

```diff
@@ -26,19 +26,45 @@ def dest_path_srs_surface_waves(filepath):
     FILE_PATTERN = re.compile(r"""
                                 IMOS_SRS-Surface-Waves_MW_
                                 (?P<platform_code>{})_FV02_
-                                (?P<latitude>[0-9]{{3}})(S|N)-
+                                (?P<latitude>[0-9]{{3}})
+                                (?P<latitude_dir>(S|N))-
                                 (?P<longitude>[0-9]{{3}})E-
                                 DM00\.nc$
                                 """.format(platforms), re.VERBOSE)
 
     file_basename = os.path.basename(filepath)
     if FILE_PATTERN.match(file_basename):
         fields = get_pattern_subgroups_from_string(file_basename, FILE_PATTERN)
+        lat = int(fields['latitude'])
+        lon = int(fields['longitude'])
+        lat_dir = fields['latitude_dir']
+
+        def coord_lower_limit_val(val):
+            """ return the bin bottom left coordinate of a coordinate value
+            exampe: 26 -> 20
+                    31 -> 30
+                    0 -> 0
+                    10 -> 10
+                    """
+            return int(round(val / 10) * 10)
```

Comment:
return int(val / 10) * 10

Reply:
This provides a more concise method of achieving this without any floating point rounding or type coercion required:

https://stackoverflow.com/a/13082705

e.g.

```python
def round_down(num, divisor=10):
    return num - (num % divisor)
```